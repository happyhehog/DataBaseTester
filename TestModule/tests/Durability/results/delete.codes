{"status":"OK","command":"SELECT * FROM t WHERE id = 1 or id = 200;","answer":"id |height    |nickname|\n1  |1.000000  |'1'     |\n200|200.000000|'200'   |"}
{"status":"OK","command":"SELECT * FROM t WHERE id = 1 or id = 200;","answer":"id |height    |nickname|\n1  |1.000000  |'1'     |\n200|200.000000|'200'   |"}
