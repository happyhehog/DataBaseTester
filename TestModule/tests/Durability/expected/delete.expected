Before restart
id |height    |nickname|
1  |1.000000  |'1'     |
200|200.000000|'200'   |

After restart
id |height    |nickname|
1  |1.000000  |'1'     |
200|200.000000|'200'   |
